### PR TITLE
refactor(types): EconCharterStartResult without runtime

### PR DESCRIPTION
**Pull request by turadg**
## Description

This defines the `EconCharterStartResult` without executing code and requiring lint suppressions.

### Security Considerations

--

### Documentation Considerations

New type, has a doc string.

### Testing Considerations

CI

### CODE REVIEW

File: packages/zoe/src/zoeService/utils.d.ts
Comment (dckc) [inline packages/zoe/src/zoeService/utils.d.ts:74]: Have you looked into parameterizing the `Instance` type such that `E(zoe).getPublicFacet(instance)` would get a useful type?

**Review by dckc — APPROVED** [review]

File: packages/zoe/src/zoeService/utils.d.ts
Comment (turadg) [reply packages/zoe/src/zoeService/utils.d.ts:74]: I had but ran into some problem I don't remember. Thanks for the prompt; I got it into this PR.  PTAAL.

Comment (turadg) [reply packages/zoe/src/zoeService/utils.d.ts:74]: @dckc I had expected Github to not allow merge without your second review because I clicked re-request review and that cleared the checkmark. 🤷

**Pull request merged**
